src/utilities/app-bridge/context.tsx

```diff
@@ -1,6 +1,6 @@
-import {createContext} from 'react';
+import React from 'react';
```

Comment:
React doesn't need to be in scope from a babel perspective but in consuming apps I was getting a typescript reference errors with react so I'm thinking this will fix it 🤞

Reply:
I wonder if this is because typescript expects React to be in scope in tsx files. As we never use any JSX would renaming these files to `.ts` fix it too?

Reply:
I thought that as well, but no 🎲